Update README.md for GT generation

smishra3 opened the pull request:
Added TIFF dependency information to the readme file. I've also provided more detailed information about the required file-naming convention for running the Fiji macro.

smishra3 committed:
Update README.md for GT generation

Added TIFF dependency information to the readme file.

pgarrison commented on Colony_mask_gt_generation/README.md:
```diff
@@ -1,6 +1,27 @@
+This method has been **validated on 3D TIFF image stacks**, where each TIFF file represents a **single z-stack** corresponding to one **timepoint**.  
+Unlike the **multi-timepoint z-stack sequences** provided in the [OME-ZARR format](https://open.quiltdata.com/b/allencell/tree/aics/emt_timelapse_dataset/), each TIFF used here contains only a single volumetric acquisition.
+
+To ensure compatibility with this Fiji macro, the original OME-ZARR files must first be **split by timepoint** and saved following the naming convention below:
```
````suggestion
To ensure compatibility with this Fiji macro, the original OME-ZARR files must first be **split by timepoint** and saved following the naming convention below:
```
Unique_Dataset_Identifier_TXX_C=2.tiff
```
````
Somewhere in these instructions we should say how to put the components together?

smishra3 replied:
addressed in d9c2dcb70d376f97d4c408f844e9ff16f4d475c5

pgarrison commented on Colony_mask_gt_generation/README.md:
```diff
@@ -1,6 +1,27 @@
+Where:  
+- **`Unique_Dataset_Identifier`** — an optional, user-defined tag to distinguish datasets or experimental conditions  
+- **`TXX`** — the timepoint index (e.g., `T01`, `T15`, etc.)  
+- **`C=2`** — denotes the imaging channel of interest (as used in this workflow)  
```
Clarify that `2` is the only value allowed by the macro?

smishra3 replied:
addressed in d9c2dcb70d376f97d4c408f844e9ff16f4d475c5

smishra3 committed:
Update README.md for curl download

Added user command for downloading checkpoint file using curl

pgarrison reviewed (dismissed):
Nice addition, thanks!

smishra3 committed:
Updated logger for eval_scale1.yaml

modified mlflow logger to csv logger for better compatibility with a wider base

smishra3 committed:
Updated logger for eval_scale2.yaml

modified logger to csv for better compatibility

smishra3 committed:
Update logger for eval_scale3.yaml

modified logger to csv logger for better compatibility

smishra3 committed:
Update README.md with bioio-ome-zarr installation information

Added bioio-ome-zarr installation instructions into the readme file

smishra3 committed:
Update README.md for gt generation

Added information for file name

smishra3 committed:
Substituted aicsimageio with bioio

Removed obsolete aicsimageio implementation with bioio

Commit:
Revert "Substituted aicsimageio with bioio"

This reverts commit d16c90e4c7d03bd9598a1af7ea6e47decbae7550.

pgarrison commented on Colony_mask_training_inference/README.md:
```diff
@@ -14,13 +14,15 @@ Then, use the following steps to create a new virtual environment and install th
    conda activate emt-acm-env
    cd Colony_mask_training_inference
    pip install .
+   pip install bioio-ome-zarr
```
We should remove the `pip install bioio-ome-zarr` lines and instead add `bioio-ome-zarr` to the environment definition. (This could be a separate PR)

smishra3 replied:
Addressed in c0c705297140e5c0c7d5cfa9a58c3d030c0638fa and in f288f71acd6b24f537859e175b4508c8203f2b63

pgarrison reviewed (changes requested):
Good addition, thanks!

smishra3 committed:
Update README.md

Removed bioio-ome-zarr explicit installation command

smishra3 committed:
Update README.md

Removed specific branch installation instruction

smishra3 committed:
Update README.md

Removed explicit bioio-ome-zarr installation instruction

smishra3 committed:
Update ColonyMask_merging_thresholding_rearranging.py

Commented out unused aicsimageio writer

pgarrison commented on Colony_mask_training_inference/README.md:
```diff
@@ -5,7 +5,6 @@
    `git clone https://github.com/AllenCell/EMT_image_analysis.git`
 2. Go inside the directory and switch to sm_colony_mask_pred_test_v0 branch.  
    `cd EMT_image_analysis`  
-   `git checkout origin/sm_colony_mask_pred_test_v0`
```
👍

Commit:
added bioio-ome-zarr installation information to pdm

Commit:
Merge branch 'pathFixreSub-v1' of https://github.com/AllenCell/EMT_image_analysis into pathFixreSub-v1

added bioio-ome-zarr installation information to pdm

smishra3 committed:
Update README.md

Removed outdated instructions
